.esh/commands/reshape_i18n_keys.py

```
@@ -0,0 +1,394 @@
+		path_map = {}
+	count = 0
+	while True:
+		clusters = find_prefix_clusters(list(obj.keys()))
+		if not clusters:
+			break
+		cluster = clusters[0]
+		prefix = cluster["prefix"]
+		members = cluster["members"]
+		preferred = container_key_for_prefix(prefix)
+
+		def on_move(old_key, new_rel, _path=path, _map=path_map):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `on_move`

Add return type annotation: `None`

(ANN202)